packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/bedrock/create-model-customization-job.ts

```diff
@@ -0,0 +1,428 @@
+      return this.props.role;
+    }
+    const stack = Stack.of(this);
+    const role = new iam.Role(this, 'BedrockRole', {
+      assumedBy: new iam.ServicePrincipal('bedrock.amazonaws.com'),
+      inlinePolicies: {
+        BedrockCreateModelCustomizationJob: new iam.PolicyDocument({
+          statements: [
+            ...(this.props.vpcConfig ? [
+              new iam.PolicyStatement({
+                actions: [
+                  'ec2:DescribeNetworkInterfaces',
+                  'ec2:DescribeVpcs',
+                  'ec2:DescribeDhcpOptions',
+                  'ec2:DescribeSubnets',
+                  'ec2:DescribeSecurityGroups',
+                ],
+                resources: ['*'],
+              }),
+              new iam.PolicyStatement({
+                actions: ['ec2:CreateNetworkInterface'],
+                resources: [
+                  stack.formatArn({
+                    service: 'ec2',
+                    resource: 'network-interface',
+                    resourceName: '*',
+                  }),
+                  stack.formatArn({
+                    service: 'ec2',
+                    resource: 'security-group',
+                    resourceName: '*',
+                  }),
+                  stack.formatArn({
+                    service: 'ec2',
+                    resource: 'subnet',
+                    resourceName: '*',
+                  }),
+                ],
+              }),
+              new iam.PolicyStatement({
+                actions: ['ec2:CreateTags'],
+                resources: [stack.formatArn({
+                  service: 'ec2',
+                  resource: 'network-interface',
+                  resourceName: '*',
+                })],
+                conditions: {
+                  StringEquals: {
+                    'ec2:CreateAction': 'CreateNetworkInterface',
+                  },
+                },
+              }),
+              new iam.PolicyStatement({
+                actions: [
+                  'ec2:CreateNetworkInterfacePermission',
+                  'ec2:DeleteNetworkInterface',
+                  'ec2:DeleteNetworkInterfacePermission',
+                ],
+                resources: ['*'],
+                conditions: {
+                  StringEquals: {
+                    'ec2:Subnet': [
+                      ...(this.props.vpcConfig
+                        ? this.props.vpcConfig.subnets.map((subnet) => subnet.subnetId)
+                        : []),
+                    ],
+                  },
+                },
+              }),
+            ] : []),
+            new iam.PolicyStatement({
+              actions: ['s3:GetObject'],
+              resources: [
+                this.s3UriToArn(this.props.trainingDataS3Uri),
+                ...(this.props.validationDataS3Uri.map((s3Uri) => this.s3UriToArn(s3Uri))),
+              ],
+            }),
+            new iam.PolicyStatement({
+              actions: ['s3:PutObject'],
```

Comment:
From the linked policy, we also need `s3:GetObject` and `s3:ListBucket` for the output bucket  (and the required bucket-level resource).